oscar_elasticsearch/search/models.py

```diff
@@ -172,7 +182,7 @@ def get_facets(cls):
                 [
                     index.FilterField("price_currency"),
                     index.SearchField("partner_sku"),
-                    index.SearchField("price_excl_tax"),
+                    index.SearchField(get_price_field_name()),
```

Comment:
Hi I think this can be fixed by just setting it to "price" and specify in setup.py the release needs at least oscar 3.0.

Reply:
But that'll break compatibility with previous versions of django-oscar.

Reply:
People that use an old version of oscar can just use the old version of django-oscar-elasticsearch as well.

Reply:
Fixed.